data_steward/cdr_cleaner/cleaning_rules/valid_death_dates.py

```diff
@@ -35,9 +35,24 @@
 # or after the current date.
 SANDBOX_INVALID_DEATH_DATE_ROWS = common.JINJA_ENV.from_string("""
 CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{sandbox_table}}` AS (
-SELECT *
-FROM `{{project_id}}.{{dataset_id}}.{{table}}`
-WHERE {{table}}_date < '{{program_start_date}}' OR {{table}}_date > {{current_date}})
+SELECT d.*
+FROM `{{project_id}}.{{dataset_id}}.{{table}}` d
+LEFT JOIN (
```

Comment:
This is a really nice query.  Consider using Jinja `-- comment --` format to describe in plain English what this inner join is accomplishing.

Reply:
Will add that, thanks